### PR TITLE
Add 3D Sonic The Hedgehog

### PR DESCRIPTION
**Pull request by fancymelonub**
Adds 3D Sonic The Hedgehog.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/3d-sonic-the-hedgehog/game.dat [hunk 1]
```diff
@@ -0,0 +1,30 @@
+title = "3D Sonic The Hedgehog"
+description = "3D Sonic The Hedgehog is a port of Sonic The Hedgehog (1991) from the Genesis to the 3DS. It includes stuff like, saving, spindash, and more!"
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
+title = "00040000000da000"
```
Comment (Leo626) [inline games/3d-sonic-the-hedgehog/game.dat]: For consistency, letters should be uppercase in title IDs.

File: games/3d-sonic-the-hedgehog/game.dat [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+region = "usa"
+release_date = "2013-12-05"
+
+[[releases]]
+title = "00040000000db500"
```
Comment (Leo626) [inline games/3d-sonic-the-hedgehog/game.dat]: Ditto.

File: games/3d-sonic-the-hedgehog/game.dat [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+region = "eur"
+release_date = "2013-12-05"
+
+[[releases]]
+title = "00040000000e9900"
```
Comment (Leo626) [inline games/3d-sonic-the-hedgehog/game.dat]: Ditto.

File: games/3d-sonic-the-hedgehog/game.dat [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+region = "jpn"
+release_date = "2013-05-15"
+
+[[testcases]]
+title = "00040000000da000"
+compatibility = "2"
+date = "2017-12-26"
+version = "HEAD-c3c684c"
+cpu = "AMD-FX 4300"
```
Comment (Leo626) [inline games/3d-sonic-the-hedgehog/game.dat]: `AMD FX-4300`

File: games/3d-sonic-the-hedgehog/game.dat [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+gpu = "AMD Radeon HD 7300 Series"
```
Comment (Hexagon12) [inline games/3d-sonic-the-hedgehog/game.dat]: Would be nice if you enter the exact GPU model using a program like _GPU-Z._

Comment (BreadFish64) [reply games/3d-sonic-the-hedgehog/game.dat]: Despite the confusion name, I believe that is the exact GPU model

Comment (Hexagon12) [reply games/3d-sonic-the-hedgehog/game.dat]: Then I would remove `Series`.

**Comment by Leo626** [comment]
Please address the comments else this will be closed.

**Comment by Hexagon12** [comment]
Closed due to inactivity.

**Pull request closed without merging**